src/Model/Entity/BussinessObject/Introduction.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Friendica\Model\Entity;
+
+use DateTime;
+use Friendica\Model\Entity\Traits\StorableTrait;
+
+/**
+ * @property int      uid
+ * @property int      fid
+ * @property Contact  contact
```

Comment:
idea: We could implement a "lazy-loading", means if we want to `get` this attribute, we automatically load the contact for this intro-entity (and log it for optimizing)